evil-types.el

```diff
@@ -244,7 +244,7 @@ the last column is excluded."
 
 (evil-define-interactive-code "c"
   "Read character."
-  (list (read-char)))
+  (list (read-char nil t)))
```

Comment:
Note that this cannot be changed as it emulates the standard interactive code 'c' which does not set the `inherit-input-method` argument, but otherwise I do not see why not to merge this.